src/main/java/dev/slimevr/vr/processor/skeleton/HumanSkeleton.java

```diff
@@ -202,8 +218,8 @@ protected HumanSkeleton(List<? extends ComputedHumanPoseTracker> computedTracker
 		// #endregion
 
 		// #region Assemble skeleton arms from chest
-		chestNode.attachChild(leftShoulderNodeHmd);
-		chestNode.attachChild(rightShoulderNodeHmd);
+		neckNode.attachChild(leftShoulderNodeHmd);
```

Comment:
Just confirming that this makes sense @Louka3000?

Reply:
Yes, `neckNode` is controlled by the physical chest tracker whereas `chestNode` is controlled by the waist tracker (or whichever one replaces it).
I made the adjustments in the bone length config (whatever it’s called) as well)